### ISSUE TITLE
Undefined method opt= for RSpec::Core::RakeTask

### ISSUE THREAD

**Issue by rchady** [strongqa/howitzer#246]
## Meta
Howitzer Version:

Driver and browser information:

Gem Versions:

## Expected Behavior
howitzer works

## Actual Behavior
Receiving the following error:

```
rake aborted!
NoMethodError: undefined method `opts=' for #<RSpec::Core::RakeTask:0x007fc1621d3550>
./test/tasks/rspec.rake:18:in `block in <top (required)>'
Tasks: TOP => default => features
(See full trace by running task with --trace)
```
## Steps to reproduce

1. Install howitzer
2. Edit config/default.yml (only made some browserstack specific changes)
3) rake

Was looking for a good place to get started, so tried to just get it working.  Not sure what is wrong at this point.

**Comment by Lec1k** [comment]
Hi @rchady,
It seems 'opts' variable was deprecated a while ago by rspec, please replace it with 'rspec_opts', like in this [example](https://github.com/strongqa/howitzer_example_rspec/blob/master/tasks/rspec.rake).

**Comment by romikoops** [comment]
@rchady if you used old version of howitzer and want to migrate fully on new one, please follow migration procedure described on wiki https://github.com/strongqa/howitzer/wiki/Migration-to-new-version
Howitzer, like Ruby on Rails consist from 2 parts: library and generated code. With first one it is easy to support compatibility on maintainers side, but generated code is on users side and we can not upgrade it automatically. Also, having generated code allows users to customize it easily regarding a project needs. I will close the issue.

**Comment by rchady** [comment]
This is a brand new installation of howitzer and a brand new project.  I was just kicking the tires to see how it works and ran in to the above issue.  I assume I am doing something wrong, was hoping you could point me in the right direction to fix it.

**Comment by romikoops** [comment]
@rchady you are right, looks like we have not updated the code in generators here https://github.com/strongqa/howitzer/blob/master/generators/rspec/templates/rspec.rake
@Lec1k Please fix

**Comment by Lec1k** [comment]
Fix in master baf0b3dcede8a252db3905590b270c917430db09
